plugin/build_baron.go

```diff
@@ -47,45 +50,78 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 		}
 		webhookConfigured := webHook.Endpoint != ""
 		if !webhookConfigured && proj.TicketCreateProject == "" {
-			return fmt.Errorf("ticket_create_project and taskAnnotationSettings.FileTicketWebHook endpoint cannot both be blank")
+			grip.Error(message.WrapError(err, message.Fields{
```

Comment:
@ablack12 do you think these errors should be grip.Critical? If it gets to this point and this is the case, we probably entered something bad into the db manually. I wouldn't want it to fail silently and only notice after a user gets frustrated.

Reply:
I think it's fine to make this critical (which means that the errors go to evergreen-ops-alerts @hadjri). 

Really all of this validation should exist for the project routes/pages instead of here; once we've implemented this for the legacy UI and routes and removed the ability to define this on the admin page, I believe we can remove this Configure loop entirely. (In my mind we would be doing this ticket after that but we may just want to make a new ticket to ensure validation for plugins is done in the right places and to remove this validation once we've removed the admin page option)

Reply:
That makes sense to me.  Since this PR's been opened and the logic for validating the build baron plugin will likely remain similar to what currently exists (just probably need to move this logic somewhere else), I've created https://jira.mongodb.org/browse/EVG-15419 and added it to the project scope.  I'll address the remaining PR comments here.